invenio_rdm_records/resources/serializers/schemas.py

```diff
@@ -79,3 +79,100 @@ def get_contributors(self, obj):
         return [
             c["person_or_org"]["name"] for c in obj["metadata"].get("contributors", [])
         ] or missing
+
+
+class DumperMixin(Schema):
```

Comment:
Seeing this I think the `FieldDumper` and `FieldLoader` classes should be ABC (interfaces). i.e. remove the _Mixin_ word from docstring and have methods be `@abstractmethod` when it makes sense.